modules/ROOT/pages/custom-data.adoc

```diff
@@ -211,6 +211,49 @@ ec validate image --image $IMAGE --public-key cosign.pub --policy "$(yq -ojson -
 There's one violation left. We could skip that one too, but I want to use that
 violation as example of how to provide custom data.
 
+== Modify the in-cluster policy
+
+The steps above make modifications to a local copy of the policy. It is also possible to create the
+policy in the cluster.
+
+The default policy is a good place to start, fetch its full form:
+
+[.console-input]
```

Comment:
I keep wondering if adding `console-input` class does anything, I did add it myself but I think that was cargo culting....

Reply:
I'm gonna wait for a braver soul to take this on.